src/structs.rs

```diff
@@ -46,6 +46,7 @@ pub struct PublicWalletData {
     pub nostr_npub: String,
     pub xprvkh: String,
     pub xpubkh: String,
+    pub xpub: String,
```

Comment:
What is this xpub?

Reply:
I will need plain xpub to build RgbWallet.

Reply:
Interesting... Good to know.